nntrainer/layers/zoneout_lstmcell.cpp

```diff
@@ -595,13 +574,8 @@ void ZoneoutLSTMCellLayer::calcGradient(RunLayerContext &context) {
     cell_state_zoneout_mask.getBatchSlice(timestep, 1);
   next_cell_state_zoneout_mask.reshape({batch_size, 1, 1, unit});
   Tensor prev_cell_state_zoneout_mask;
-  if (!test) {
-    prev_cell_state_zoneout_mask =
-      next_cell_state_zoneout_mask.zoneout_mask(cell_state_zoneout_rate);
-  } else {
-    next_cell_state_zoneout_mask.multiply(-1.0f, prev_cell_state_zoneout_mask);
-    prev_cell_state_zoneout_mask.add_i(1.0f);
-  }
+  next_cell_state_zoneout_mask.multiply(-1.0f, prev_cell_state_zoneout_mask);
```

Comment:
consider something like `prev_hidden_state_zoneout_mask = next_hidden_state_zoneout_mask.apply([](float x) {return x <= epsilon;});` - it will be faster.

Reply:
Done